packages/libsql-client/src/hrana.ts

```diff
@@ -255,7 +256,7 @@ export async function executeHranaBatch(
     disableForeignKeys: boolean = false,
 ): Promise<Array<ResultSet>> {
     if (disableForeignKeys) {
-        batch.step().run("PRAGMA foreign_keys=off")
+        batch.step().run("PRAGMA foreign_keys=off");
```

Comment:
Diff noise.

Reply:
Prettier ran on commit, have the rules changed?